### PR TITLE
Non default

### PR DESCRIPTION
**Pull request by double-beep**
New commit

### CODE REVIEW

**Comment by double-beep** [comment]
Just for completing this pull request.

**Comment by helper-beep** [comment]
Ok.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 Copyright 2018
 Batch experiments
+Assigned to sb.
+No contributors needed.
+Batch experiments starting soon.
```
Comment (helper-beep) [inline README.md]: Required changes.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,5 @@
 Copyright 2018
 Batch experiments
```
Comment (helper-beep) [inline README.md]: Copyright R

**Comment by helper-beep** [comment]
ok.

**Pull request merged**